portfolio/src/main/java/com/google/sps/servlets/ServiceServlet.java

```diff
@@ -0,0 +1,38 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+/** Servlet responsible for handling memes. */
+@WebServlet("/service-handler")
+public class BookNowServlet extends HttpServlet {
```

Comment:
I think the class name should be ServiceServlet instead of BookNowServlet?

Reply:
yessirrr thanksss